pkg/credentialschema/opts.go

```diff
@@ -177,7 +190,11 @@ func processValidatedOpts(opts *resolveOpts) ([]*verifiable.Credential, *issuer.
 		metricsLogger = opts.metricsLogger
 	}
 
-	issuerMetadata, err := processIssuerMetadataOpts(&opts.issuerMetadataSource, metricsLogger)
+	if opts.httpClient == nil {
```

Comment:
Instead of requiring an HTTP client, can we use `http.DefaultClient` by default?